### ISSUE TITLE
Api Version 2 Migration

### ISSUE THREAD

**Issue by CapitanRedBeard** [bugsnag/bugsnag-react-native#233]
### Description

Just got this email from the BugSnag team:
```
Action required: All users of data access API version 1

In one month, May 10th, we'll be changing the default version on our data access API from v1 to v2. To continue using v1 of the API after this date, you will need to specify the version in a header: "X-Version: 1".

We’d also like to inform you that we are planning to shutdown API v1 in three months on July 10th. We'll send a reminder closer to this date, but we would encourage you to switch to v2 as soon as possible. For instructions on migrating, please visit our docs. 

Dates to remember:
April 10th: First notice 
May 10th: Default API version changes from v1 to v2
July 10th: API v1 deprecated 
The data access API provides a flexible interface for manipulating your data to view, filter, aggregate, and take other actions on errors in Bugsnag. v2 of the Bugsnag API has been available since June 2017, and you can learn more about what can be done with the API in our docs. 

APIs for reporting errors, session, builds, and uploading files are unaffected by this change. 

If you have any questions about this announcement or need any help, please contact us.

- The Bugsnag Team 
```

*TL;DR*: Upgrading from API v1 to v2. v1 will be deprecated on July 10th

I'm exclusively using this library to interface with the BugSnag API, so my question is are we already in place to point to the new API? It seems like they are choosing API v2 to be the new default api, so I wouldn't think any endpoints have to change, however with a new API sometimes comes new endpoint contracts so I just want to be sure that if we do need to modify something we do it as soon as possible.

Cheers!

**Comment by kattrali** [comment]
This email is related to the [Data Access API](https://docs.bugsnag.com/api/data-access/), which is an interface for downloading your data from Bugsnag. Notifier libraries such as `bugsnag-react-native` are unaffected as they use the write-only [Error reporting API](https://docs.bugsnag.com/api/error-reporting/).

**Comment by CapitanRedBeard** [comment]
Ahhh thanks @kattrali

**Comment by kattrali** [comment]
No problem! Nothing to worry about 😄
